folders.py

```diff
@@ -45,7 +45,7 @@ def parse(file_name, user_id):
         data = data_file.read()
 
     # spotify:playlist, spotify:start-group, spotify:end-group
-    rows = re.split(b'spotify:[pse]', data)
+    rows = re.split(r'spotify:(?=[pse])', data)
```

Comment:
You want to keep the `b` prefix, since we're dealing with bytestrings here. You can add the `r` too, but it's optional here, since we don't have any backslashes.